built/History.js

```diff
@@ -4,22 +4,22 @@ var __extends = (this && this.__extends) || function (d, b) {
     function __() { this.constructor = d; }
     d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
 };
-var React = require('react');
```

Comment:
Yep, we discussed it with @billba today and it seems there is a plan for not integrating build files in the PR. This would also highly reduce the merge friction between PRs.

I ll only PR source files in subsequent PRs.